src/main.ts

```diff
@@ -1,16 +1,19 @@
 import { NestFactory } from '@nestjs/core';
-import { ConfigService } from '@nestjs/config';
 import { ValidationPipe } from '@nestjs/common';
-
+import * as dotenv from 'dotenv';
 import { AppModule } from './app.module';
 
+dotenv.config();
+
 async function bootstrap() {
+  const PORT = process.env.PORT || 8080;
   const app = await NestFactory.create(AppModule);
-  const config = app.get(ConfigService);
 
   app.useGlobalPipes(new ValidationPipe());
 
```

Comment:
![image](https://user-images.githubusercontent.com/67024726/121892238-2600c200-cd25-11eb-86d5-93235eee3356.png)
Your main.ts file should look like this.
Add all files with user/posts/auth  to gitignore file

Reply:
Add to gitignore, please.